<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioWidget</class>
 <widget class="QWidget" name="RadioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="unicorn::StackedWidget" name="stackedWidget">
     <widget class="QWidget" name="mainPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QuickStartWidget" name="quickStartWidget">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="nowPlayingFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="splitter_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lastStationLabel">
            <property name="text">
             <string>Last Station</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlayableItemWidget" name="lastStation">
            <property name="text">
             <string>A Radio Station</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="splitter_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="personalLabel">
         <property name="text">
          <string>Personal Stations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="personalFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="PlayableItemWidget" name="library">
            <property name="text">
             <string notr="true">My Library Radio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlayableItemWidget" name="mix">
            <property name="text">
             <string notr="true">My Mix Radio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlayableItemWidget" name="rec">
            <property name="text">
             <string notr="true">My Recommendations Radio</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="splitter_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="networkLabel">
         <property name="text">
          <string>Network Stations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="networkFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="PlayableItemWidget" name="friends">
            <property name="text">
             <string notr="true">My Friends' Radio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlayableItemWidget" name="neighbours">
            <property name="text">
             <string notr="true">My Neighbourhood Radio</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="splitter">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="recentLabel">
         <property name="text">
          <string>Recent Stations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="recentStations">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="spinnerPage">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="spinner">
         <property name="text">
          <string notr="true">TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nonSubPage">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="nonSubFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="title">
            <property name="text">
             <string>Subscribe to listen to radio</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="headphones">
              <property name="text">
               <string notr="true">Headphones</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="description">
            <property name="text">
             <string>You need to be a Last.fm subscriber to listen to radio in this app. Subscribe now to start listening and take advantage of other great benefits too!</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PushButton" name="subscribe">
            <property name="text">
             <string>Subscribe to Last.fm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PushButton" name="listen">
            <property name="text">
             <string>Listen free on www.last.fm</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QuickStartWidget</class>
   <extends>QFrame</extends>
   <header>../Widgets/QuickStartWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayableItemWidget</class>
   <extends>QPushButton</extends>
   <header>../Widgets/PlayableItemWidget.h</header>
  </customwidget>
  <customwidget>
   <class>unicorn::StackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>lib/unicorn/widgets/StackedWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>../Widgets/PushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
